base/applications/notepad/help/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+# NOTE: The filenames of help document must be globally unique.
+#       If you want to add image files, please add 'notepad-' prefix to the image filename.
+
+# res/*
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/res/notepad.js DESTINATION reactos/Help/res FOR bootcd)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/res/notepad.css DESTINATION reactos/Help/res FOR bootcd)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/res/notepad-32x32.png DESTINATION reactos/Help/res FOR bootcd)
+
+# *.txt
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/notepad_en-US.txt DESTINATION reactos/Help FOR bootcd) # English
+
+# *.html
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/notepad_en-US.html DESTINATION reactos/Help FOR bootcd) # English
```

Comment:
Consider also the alternative of having all the help files inside e.g. `reactos/Help/en-US/...` , which is something that goes along the line with what is done also with MUI files, etc.

Reply:
Directories in ISO filesystem are limited resource. We cannot use arbitrary directory names.

Reply:
> Directories in ISO filesystem are limited resource.

Don't we have reactos.cab for that matter os it has way other use case?

Reply:
Yep, we use cab. Sorry.

The number of major languages in ReactOS is less than 30. We will create a few of ten directories if so.

Reply:
> Yep, we use cab. Sorry.
> 
> The number of major languages in ReactOS is less than 30. We will create a few of ten directories if so.

Then they could be easily fit into cabinet, I guess and ISO will remain unaffected directory-wise.

Reply:
The flat directory is simpler manage than the tree directories. It won't take time by choosing directory path. Making 15 or more directories seems stupid to me.